lib/rules/no-multi-spaces.js

```diff
@@ -93,7 +93,7 @@ module.exports = {
                 const sourceCode = context.getSourceCode(),
                     source = sourceCode.getText(),
                     allComments = sourceCode.getAllComments(),
-                    pattern = /[^\n\r\u2028\u2029\t ].? {2,}/g;  // note: repeating space
+                    pattern = /[^\s].? {2,}/g;  // note: repeating space
```

Comment:
The new regEx is a bit more restrictive then before. It will include tabs and spaces on top of the linebreaks in the "not" group. Is this intentional?